main/menus.js

```diff
@@ -201,6 +201,17 @@ const appMenuTemplate = [
       {
         type: 'separator'
       },
+      {
+        label: 'Clean temporary files',
```

Comment:
From discussion on #762 , sounds like some people want to clean up old recordings before the system clean-up runs

Reply:
Can't we just remove the files when Kap exits normally?

Reply:
Hmm, I guess we could. I was thinking about this as if you don't have an issue with space, we could show these recordings on the exports window. Not at the moment, but when the editor window is reworked to also show the export, the exports window could become history of sorts, and it would persist across relaunch as long as the temp files were still there.

That would be the same as now (we don't clean the temp files) but we would just keep track of them

Reply:
For what purpose though? If the user is done with it, they'll want it deleted. If they're not, they'll rather want it in a safe place that will not be cleaned up at a random time.

Reply:
Updated